charts/rancher-webhook/rancher-webhook/1.0.0+up0.2.0-beta01/Chart.yaml

```diff
@@ -5,11 +5,11 @@ annotations:
   catalog.cattle.io/os: linux
   catalog.cattle.io/release-name: rancher-webhook
 apiVersion: v2
-appVersion: 0.2.0-alpha04
+appVersion: 0.2.0-beta01
```

Comment:
Are we moving from beta to a 0.2.0 before release ? Or sticking with beta in release ? Also is there a reason we aren't just releasing a 1.0.0 ?

Reply:
I'm not aware of the version decisions for the webhook.

@ryansann or @dramich: can either of you answer these questions?

Reply:
I'll defer to @dramich on this. But I do know we have been using the v0.1.x line for 2.5 charts and v0.2.x line for 2.6 charts

Reply:
The goal is to release 0.2.0 before (or with) 2.6 releases, hence moving this to beta now. 0.1.0 is 2.5 and 0.2.0 is 2.6 line.